modules/util/css-builder/src/main/resources/maven-wrapper/.mvn/wrapper/maven-wrapper.properties

```diff
@@ -0,0 +1 @@
+distributionUrl=https://repo1.maven.org/maven2/org/apache/maven/apache-maven/3.5.0/apache-maven-3.5.0-bin.zip
```

Comment:
We don't want the maven wrapper in `src/main/resources` since this is only for the test.  In this case, this file should be in `src/test/resources`